src/applications/mhv-secure-messaging/components/Dashboard/WelcomeMessage.jsx

```diff
@@ -49,7 +49,8 @@ const WelcomeMessage = () => {
           target="_blank"
           rel="noreferrer"
         >
-          Go back to the previous version of secure messaging
+          Go back to the previous version of secure messaging (opens in a new
+          tab)
```

Comment:
Looks like the ucd screenshots say "opens in new tab", but yours include "a".  I think that "a" should be removed.